src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -150,7 +177,8 @@ class LatticeConfigurationDialog extends React.Component {
                                 <label className="fg-label">Lattice 'a'</label>
                                 <div className="fg-line">
                                     <input type="number" name="a" className="form-control fc-alt fg-input" min="0"
-                                        step="0.05" disabled={this.isDisabled('a')}
+                                        disabled={this.state.isNonPeriodic}
```

Comment:
We should not be changing this but rather adjust the logic for `this.isDisabled`